tasks/agent.py

```diff
@@ -59,6 +59,7 @@
 
 PUPPY_CORECHECKS = [
```

Comment:
should also be added in `tasks/android.py`  `ANDROID_CORECHECKS` to be loaded in android.  Not sure if there's a better way to make this easily supported cross platform, as there are some `core` checks not used by android